src/app/features/ui/filter-search-button/dropdown/event-dropdown.component.html

```diff
@@ -0,0 +1,165 @@
+<menu class="dropdown">
+    <div class="dropdown__section">
+        <p class="text-body">DATA UTWORZENIA</p>
+        <div
+            (click)="toggleFiltrRadio(creationDate1)"
+            class="dropdown__section__holder">
+            <input
+                #creationDate1
+                name="creationDate"
+                type="radio"
+                value="{{ creationDate1.checked === true ? 'Dzisiaj' : '' }}"
+                id="creation-date-today"
+                class="dropdown__section__holder__filtr-option" />
+            <label for="creation-date-today">Dzisiaj</label>
+        </div>
+        <div
+            class="dropdown__section__holder"
+            (click)="toggleFiltrRadio(creationDate2)">
```

Comment:
Mała sprawa, ale jeśli się trzymamy nazw w języku angielskim to wszędzie zamiast `filtr` powinno być `filter`

Reply:
poprawiłem